### ISSUE TITLE
Suggestions on processing chat records of long conversation records

### ISSUE THREAD

**Issue by suwubee** [labring/FastGPT#820]
**例行检查**

[//]: # '方框内填 x 表示打钩'

- [x] 我已确认目前没有类似 features
- [x] 我已确认我已升级到最新版本
- [x] 我已完整查看过项目 README，已确定现有版本无法满足需求
- [x] 我理解并愿意跟进此 features，协助测试和提供反馈
- [x] 我理解并认可上述内容，并理解项目维护者精力有限，**不遵循规则的 features 可能会被无视或直接关闭**

**功能描述**
1，并行逻辑的回答混乱和聊天记录丢失
如果一个动作后同时触发两个回复模块，并未设置回复的触发顺序，实际回复中会夹杂两条回复，建议以优先响应的回复完了再回复另一个（事实上这两个模块顺序不重要，但是如果设置触发顺序，会慢很多）。
如果指定回复后再触发AI对话，AI对话中不会记录指定回复的聊天内容，或者并行两个AI回复，总是会丢一个聊天记录。

2，建议设置每个涉及到聊天记录保存的功能
是否保存到聊天记录，是否精简后保存到聊天记录，精简规则可以设置另起一个AI模块来处理后单独插入，或者是js脚本正则提取，或者是只提取前面多少的内容

3，回复聊天记录的模块

**应用场景**
在长对话（15+）和并行流程的处理中，聊天记录的处理与否直接影响整个流程的效率和AI回复的精准度。

**相关示例**

**Comment by c121914yu** [comment]
> Bot detected the issue body's language is not English, translate it automatically. 👯👭🏻🧑‍🤝‍🧑👫🧑🏿‍🤝‍🧑🏻👩🏾‍🤝‍👨🏿👬🏿         
----    
**Routine inspection**

[//]: # 'Fill x in the box to indicate tick'

- [x] I have confirmed that there are currently no similar features
- [x] I have confirmed that I have upgraded to the latest version
- [x] I have fully reviewed the project README and determined that the existing version cannot meet the needs.
- [x] I understand and am willing to follow up on this feature, assist with testing and provide feedback
- [x] I understand and agree with the above content, and understand that project maintainers have limited energy. **Features that do not follow the rules may be ignored or closed directly**

**Function description**
1. Parallel logic answers are confusing and chat records are lost
If two reply modules are triggered at the same time after an action, and the triggering order of the replies is not set, the actual reply will be mixed with two replies. It is recommended to reply with the priority response before replying to the other one (in fact, the order of these two modules is not important. But if you set the trigger sequence, it will be much slower).
If an AI conversation is triggered after specifying a reply, the chat content of the specified reply will not be recorded in the AI ​​conversation, or if two AI replies are made in parallel, a chat record will always be lost.

2. It is recommended to set up every function related to chat record saving.
Whether to save to the chat record, whether to save to the chat record after streamlining, the streamlining rules can be set up with another AI module to process and then inserted separately, or it can be extracted by regular js script, or only the previous content can be extracted.

3. Module for replying to chat records

**Application Scenario**
In the processing of long conversations (15+) and parallel processes, whether the chat records are processed or not directly affects the efficiency of the entire process and the accuracy of AI responses.

**Related Examples**
